src/nft/utils/listNfts.ts

```diff
@@ -37,11 +37,12 @@ export const ListingMarkets: ListingMarket[] = [
     fee: 0.5,
     icon: '/nft/svgs/marketplaces/x2y2.svg',
   },
-  {
-    name: 'LooksRare',
-    fee: 1.5,
-    icon: '/nft/svgs/marketplaces/looksrare.svg',
-  },
+  // Hiding LooksRare until we can support their v2 SDK
+  // {
+  //   name: 'LooksRare',
+  //   fee: 1.5,
+  //   icon: '/nft/svgs/marketplaces/looksrare.svg',
+  // },
```

Comment:
We would still need to update the logic to support v2, all our calls to list to LR v1 would fail

Reply:
Expanding on this, why a feature gate wouldn't be useful in this scenario is that when the gate is opened it would just be to a broken feature so we'd never have a reason to open the gate. 

If we'd like to add a feature gate when we get around to working on supporting v2 that's a different story but that addition should be kept to when the work for supporting v2 actually starts

Reply:
They will be uncommented but we'll need to make major changes to our listing logic as well as the app configuration (Node 16 vs 14) before we can

Reply:
Until then, to prevent users from getting a broken experience when they try to list to LooksRare starting tomorrow, we can hide the marketplace by making this code change.

Reply:
i do think feature gating marketplaces may be a useful feature for future cases, and generally we should avoid committing commented code. but in this case given the timeline i'm okay merging this as is since it doesn't introduce risk in any other way

Reply:
@cmcewen @just-toby Personally I prefer keeping in commented code as a reference for when this specific codeblock is re-added it will be nearly identical save the fee, but given team preference I've removed it.